engine/lite/k8sDeploymentManager/src/main/scala/pl/touk/nussknacker/k8s/manager/K8sDeploymentManager.scala

```diff
@@ -52,7 +52,9 @@ case class K8sDeploymentManagerConfig(dockerImageName: String = "touk/nussknacke
                                       dockerImageTag: String = BuildInfo.version,
                                       scalingConfig: Option[K8sScalingConfig] = None,
                                       configExecutionOverrides: Config = ConfigFactory.empty(),
-                                      k8sDeploymentConfig: Config = ConfigFactory.empty())
+                                      k8sDeploymentConfig: Config = ConfigFactory.empty(),
+                                      releaseName: Option[String] = None
```

Comment:
release name is helm specific word. I wonder if it is a good option to see here technical detail of fact that nussknacker is bundled by helm. What will happen if we replace helm by sth else? Maybe it should be renamed to `objectNamePrefix`? WDYT? Also maybe it should be at the beginning. It looks like first thing that should be configured